bin/lib/check_reqs.js

```diff
@@ -246,7 +273,11 @@ module.exports.check_android_target = function(originalError) {
     //   Google Inc.:Glass Development Kit Preview:20
     var valid_target = module.exports.get_target();
     var msg = 'Android SDK not found. Make sure that it is installed. If it is not at the default location, set the ANDROID_HOME environment variable.';
-    return tryCommand('android list targets --compact', msg)
+    //   Changing "targets" to "target" is stupid and makes more code.  Thanks Google!
+    var cmd = 'android list targets --compact';
+    if(forgivingWhichSync('avdmanager').length > 0)
+      cmd = 'avdmanager list target --compact';
```

Comment:
I just realized this but I think what we want here is to use `sdkmanager` and not `avdmanager`, right @infil00p ? `avdmanager list targets` returns either AVD device definitions or existing AVD IDs.